mysql-replication/mainTemplate.json

```diff
@@ -40,7 +40,7 @@
         },
         "vmSize": {
             "type": "string",
```

Comment:
@liupeirong We're not setup to consume mainTemplate.json outside of marketplace. This is going to break a bunch of testing/validation scenarios. Can you please rename these back to azuredeploy.json? Thanks!